src/poliastro/maneuver.py

```diff
@@ -60,41 +58,159 @@ def __getitem__(self, key):
+        dv_b = np.array([0, -dv_b.decompose().value, 0]) * u.m / u.s
+
+        # Transform to IJK frame
+        dv_a = pqw2ijk(dv_a, orbit_i.inc, orbit_i.raan, orbit_i.argp) * u.m / u.s
+        dv_b = pqw2ijk(dv_b, orbit_i.inc, orbit_i.raan, orbit_i.argp) * u.m / u.s
+
+        t_trans = np.pi * np.sqrt(a_trans ** 3 / k)
+
         return cls((0 * u.s, dv_a), (t_trans, dv_b))
 
     @classmethod
     def bielliptic(cls, orbit_i, r_b, r_f):
-        """Compute a bielliptic transfer between two circular orbits.
+        r"""Compute a bielliptic transfer between two circular orbits.
+
+        The bielliptic maneuver employs two Hohmann transfers, therefore two
+        intermediate orbits are stablished. We define the different radius
+        relationships as follows:
+
+        .. math::
+            \begin{align}
+                a_{trans1} &= \frac{r_{i} + r_{b}}{2}\\
+                a_{trans2} &= \frac{r_{b} + r_{f}}{2}\\
+            \end{align}
+
+        The increments in the velocity are:
 
+        .. math::
+            \begin{align}
+                \Delta v_{a} &= \sqrt{\frac{2\mu}{r_{i}} - \frac{\mu}{a_{trans1}}} - v_{i}\\
+                \Delta v_{b} &= \sqrt{\frac{2\mu}{r_{b}} - \frac{\mu}{a_{trans2}}} - \sqrt{\frac{2\mu}{r_{b}} - \frac{\mu}{a_trans{1}}}\\
+                \Delta v_{c} &= \sqrt{\frac{\mu}{r_{f}}} - \sqrt{\frac{2\mu}{r_{f}} - \frac{\mu}{a_{trans2}}}\\
+            \end{align}
+
+        The time of flight for this maneuver is the addition of the time needed for both transition orbits, following the same formula as
+        Hohmann:
+
+        .. math::
+            \begin{align}
+                \tau_{trans1} &= \pi \sqrt{\frac{a_{trans1}^{3}}{\mu}}\\
+                \tau_{trans2} &= \pi \sqrt{\frac{a_{trans2}^{3}}{\mu}}\\
+            \end{align}
+
+        Parameters
+        ----------
+        orbit_i: poliastro.twobody.orbit.Orbit
+            Initial orbit
+        r_b: astropy.unit.Quantity
+            Altitude of the intermediate orbit
+        r_f: astropy.unit.Quantity
+            Final altitude of the orbit
         """
-        r_i = orbit_i.a
-        v_i = orbit_i.v
+        if orbit_i.nu is not 0 * u.deg:
+            orbit_i = orbit_i.propagate_to_anomaly(0 * u.deg)
+
+        # Initial orbit data
         k = orbit_i.attractor.k
-        R = r_f / r_i
-        Rs = r_b / r_i
-        dv_a = ((np.sqrt(2 * Rs / (1 + Rs)) - 1) * v_i).decompose()
-        dv_b = (
-            -np.sqrt(2 / Rs) * (np.sqrt(1 / (1 + Rs / R)) - np.sqrt(1 / (1 + Rs))) * v_i
-        ).decompose()
-        dv_c = (-(np.sqrt(2 * Rs / (R + Rs)) - 1) / np.sqrt(R) * v_i).decompose()
-        t_trans1 = (np.pi * np.sqrt((r_i * (1 + Rs) / 2) ** 3 / k)).decompose()
-        t_trans2 = (np.pi * np.sqrt((r_i * (R + Rs) / 2) ** 3 / k)).decompose()
+        r_i = orbit_i.r
+        v_i = orbit_i.v
+        h_i = norm(cross(r_i.to(u.m), v_i.to(u.m / u.s)) * u.m ** 2 / u.s)
```

Comment:
```suggestion
        h_i = norm(cross(r_i.to(u.m), v_i.to(u.m / u.s)))
```

Reply:
I tried to supress those units as suggested but then the tests did not passed, since the following exception is raised:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-15-1ae6b7d64845> in <module>
      1 orbit_i = Orbit.circular(Earth, alt=191.34411 * u.km, inc=0*u.deg)
----> 2 hohmann(orbit_i, r_f=35781.34857*u.km + Earth.R)

<ipython-input-14-a8a7f89cc4a4> in hohmann(orbit_i, r_f)
     29 
     30     #h_i = norm((cross(r_i.to(u.m), v_i.to(u.m / u.s)))) * u.m ** 2 / u.s
---> 31     h_i = norm(cross(r_i.to(u.m), v_i.to(u.m / u.s)))
     32     print(h_i)
     33     p = h_i ** 2 / k.to(u.m**3 / u.s**2)

~/Documentos/Github/poliastro/src/poliastro/util.py in norm(vec)
     79 
     80     """
---> 81     return norm_fast(vec.value) * vec.unit
     82 
     83 

AttributeError: 'numpy.ndarray' object has no attribute 'value'
```

Reply:
Oops, perhaps `* u.m ** 2 / u.s` has to be _outside_ of the parentheses?

Reply:
There we go! The thing was with the `core` functions. Do you think it would be interesting to implement them `(rv_pqw, pqw2ijk, cross)` in the high level api, like for example the `util.norm` function?

Reply:
Maybe! Let's leave that for a future PR :)